tests/nn/data_parallel/test_fsdp_overlap.py

```diff
@@ -143,53 +173,48 @@ def run(compute_cycles, all_gather_mb):
             cpu_iter.add(cpu_iter_time)
             cpu_wait.add(cpu_wait_for_gpu_time)
             gpu_compute.add(sum(times))
-            gpu_total.add(e1.elapsed_time(e2))
+            gpu_total.add(overall_gpu_time)
```

Comment:
after this change, it seems that CI shows the resulting gpu timing became around 1~2 ms, which is strange since we asked for 100ms and there are 4 layers, should be around 400ms (if perfectly overlapped). Am I misreading the test results?